frontend/src/lib/components/AppEditorLink/AppEditorLink.tsx

```diff
@@ -40,7 +40,7 @@ export function AppEditorLink({
                 footer={<Button onClick={() => setModalOpen(false)}>Close</Button>}
                 onCancel={() => setModalOpen(false)}
             >
-                <AuthorizedUrls actionId={actionId} pageKey="app-editor-link" />
+                <AuthorizedUrlList actionId={actionId} type={AuthorizedUrlListType.TOOLBAR_URLS} />
```

Comment:
I pushed some changes to remove the `pageKey` as it was confusing and arguably not very helpful